libraries/shared/factories/storage/transformer.go

```diff
@@ -51,5 +51,17 @@ func (transformer Transformer) Execute(row utils.StorageDiffRow) error {
 	if decodeErr != nil {
 		return decodeErr
 	}
-	return transformer.Repository.Create(row.BlockHeight, row.BlockHash.Hex(), metadata, value)
+
+	if metadata.Type == utils.PackedSlot {
+		for _, v := range value.([]string) {
```

Comment:
Not sure if iterating through the values here makes sense, or not. At the moment, the way it's implemented we lose the ability to know which field the value belongs to. So I may rework this to pass in a slice of strings, and then require the individual implementations of `Create` deal with it.